Sofa/framework/Helper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -760,4 +760,12 @@ const std::map< std::string, Renamed, std::less<> > renamedComponents = {
 
 };
 
+
+const std::map< std::string, Dealiased, std::less<> > dealiasedComponents = {
```

Comment:
```suggestion
const std::map< std::string, Dealiased, std::less<> > dealiasedComponents = {
```

Just curious if `std::less<>` is required here

Reply:
It can be ready now, as `std::less` is also used in the other maps. I was just curious